auth/auth_test.go

```diff
@@ -38,3 +41,95 @@ func TestIsTokenExpired(t *testing.T) {
 		assert.Equal(t, tt.expected, got)
 	}
 }
+
+type authConfigManager struct {
```

Comment:
Shouldn't this go to `test/mocks`, according to new guidelines?

Reply:
It should, but the guideline isn't merged yet :smile:

Reply:
It's not strict requirement to have every mock in there, I guess. That would be reasonable for bigger mocks, imo.